src/tmpfiles/tmpfiles.c

```diff
@@ -290,14 +290,18 @@ static int log_unresolvable_specifier(const char *filename, unsigned line) {
          * not considered as an error so log at LOG_NOTICE only for the first time
          * and then downgrade this to LOG_DEBUG for the rest. */
 
+        int log_level = notified || arg_root || sd_booted() == 0 ?
+                LOG_DEBUG : LOG_NOTICE;
```

Comment:
please add discussion of the sd_booted() check into comments here...

why not use a running_in_chroot() check here?